src/main/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusValidation.java

```diff
@@ -16,39 +16,60 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.io.Serializable;
 import java.util.Arrays;
 import java.util.Collection;
+import java.util.Date;
 import java.util.Enumeration;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
 
-public class GtfsPlusValidation {
-    public static final Logger LOG = LoggerFactory.getLogger(GtfsPlusValidation.class);
+/** Generates a GTFS+ validation report for a file. */
+public class GtfsPlusValidation implements Serializable {
+    private static final long serialVersionUID = 1L;
+    private static final Logger LOG = LoggerFactory.getLogger(GtfsPlusValidation.class);
     private static final FeedStore gtfsPlusStore = new FeedStore(DataManager.GTFS_PLUS_SUBDIR);
     private static final String NOT_FOUND = "not found in GTFS";
 
+    // Public fields to appear in validation JSON.
+    public final String feedVersionId;
+    /** Indicates whether GTFS+ validation applies to user-edited feed or original published GTFS feed */
+    public boolean published;
+    public long lastModified;
+    /** Issues found for this GTFS+ feed */
+    public List<ValidationIssue> issues = new LinkedList<>();
```

Comment:
Just curious, why an is a LinkedList used here?

Reply:
Not really sure... that was the type used previously in the method `validateGtfsPlus`, so I just kept it.